src/endpoints.ts

```diff
@@ -0,0 +1,25 @@
+import * as https from 'https';
```

Comment:
Okay. The key there would be making sure that the entry point that frontend2 hits doesn't include `https` in its dependency graph. In theory tree-shaking could prune it even if that's not the case, but it's got enough caveats that I don't trust it 😛